bin/boss-account.py

```diff
@@ -108,18 +109,18 @@ def create(self):
             'Namespace': 'AWS/Billing',
             'Statistic': 'Maximum',
             'Dimensions': [{'Name': 'Currency', 'Value': currency}],
-            'Period': 10,
+            'Period': 21600,
```

Comment:
Why change this, what is it doing? (I just copied the original code so I'm not sure what it is setting)

Reply:
First off it fails with 10 now.  I think we tried at one time to change it to get better resolution then 6 hours.  changing it to a lower number doesn't actually change the how often they can fire. what ends up happening is they all reset if it is less then 6 hours (21600).  So we end  up getting all the emails again.

Reply:
Maybe a comment that this value cannot be less than 6hrs (or something)
Should this be an optional argument?